galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -148,3 +148,18 @@ messages:
         type: string
       - name: destPorts
         type: "[]int"
+
+  - name: "JwtFailureDueToInvalidServicePortPrefix"
```

Comment:
I tend to think this should be in a separate PR and cherry-picked for 1.4 and 1.5. Then we'll ensure all our releases don't accidentally reserve this number in the future.

Reply:
See https://github.com/istio/istio/pull/20671/files

Reply:
I am a bit confused. Why does this not apply to master? Sure, in master it should say "use the new policy", but don't we still want to find issues with old policies if users have them?

Reply:
I think we should keep message numbers consistent across versions if analyzers are added to an already released branch, but new analyzers have been added to master. 113 has already been used in master (https://github.com/istio/istio/blob/master/galley/pkg/config/analysis/msg/messages.yaml#L149) up to 118.

We are setting a precedent for how numbers are being used and this would help PRs against istio.io documentation as well.

Reply:
@howardjohn I'm with you that we should keep it in master. If folks are not using the old APIs there will be no messages and if they are I would hope we help them out via the analyzer messages. That's the reason Jacob had submitted the PR on master initially.